lib/SILOptimizer/Utils/SILInliner.cpp

```diff
@@ -559,15 +559,18 @@ SILValue SILInlineCloner::borrowFunctionArgument(SILValue callArg,
       || callArg.getOwnershipKind() != ValueOwnershipKind::Owned) {
     return callArg;
   }
-  auto *borrow = getBuilder().createBeginBorrow(AI.getLoc(), callArg);
+
+  SILBuilderWithScope beginBuilder(AI.getInstruction(), getBuilder());
+  auto *borrow = beginBuilder.createBeginBorrow(AI.getLoc(), callArg);
   if (auto *tryAI = dyn_cast<TryApplyInst>(AI)) {
-    SILBuilder returnBuilder(tryAI->getNormalBB()->begin());
+    SILBuilderWithScope returnBuilder(tryAI->getNormalBB()->begin());
```

Comment:
Can you pass in the builder to /all/ of these so they all use the same builder context?

Reply:
Wouldn't that change the semantic though? The other builders didn't do that before.

Reply:
That would. And I am saying as the person who wrote this code that it was probably wrong.

Reply:
Missed your comment. Doing this as a follow-up.